tools/image_processing/imagej2/imagej2_noise/imagej2_noise.py

```diff
@@ -0,0 +1,100 @@
+# Galaxy wrapper for use with Imagej2 by Greg Von Kuster
+"""
```

Comment:
@gregvonkuster I'm wondering if it wouldn't be easier to put this all into cheetah. You would save a lot of boilerplate generating the argparse.